.github/workflows/test-full-sync.yml

```diff
@@ -221,6 +221,20 @@ jobs:
           --ssh-flag="-o ServerAliveInterval=5" \
           --command="docker logs --follow ${{ env.CONTAINER_NAME }}"
 
+          EXIT_CODE=$(\
+          gcloud compute ssh \
+          sync-checkpoint-${{ env.GITHUB_REF_SLUG_URL }}-${{ env.GITHUB_SHA_SHORT }} \
+          --zone ${{ env.ZONE }} \
+          --quiet \
+          --ssh-flag="-o ServerAliveInterval=5" \
+          --command="docker wait ${{ env.CONTAINER_NAME }}")
+
+          if [[ ${EXIT_CODE} -eq "0" ]]; then
+            exit 0
+          else
+            exit 1
+          fi
```

Comment:
Can we do just `exit ${EXIT_CODE}`?

Reply:
Fixed with [imp(ci): use single line exit](https://github.com/ZcashFoundation/zebra/pull/3968/commits/d6c37c167384d92297dbe87dcc1ddeef8303fdf0)